Makefile

```
@@ -120,6 +120,7 @@ help:
 	@echo " - vet               examines Go source code and reports suspicious constructs"
 	@echo " - test              run unit test"
 	@echo " - test-sqlite       run integration test for sqlite"
+	@echo " - pr#PRID           build and start gitea from a PR with integration test data loaded"
```

Maybe something similar to this instead? Or could use `[]` or `{}` I'm not sure what's preferred or best practice.
```suggestion
	@echo " - pr#<index>        build and start gitea from a PR with integration test data loaded"
```